test_rclcpp/test/test_signal_handler_before_shutdown__expected_output.txt

```diff
@@ -1,7 +1,6 @@
 Registered custom signal handler.
 Called rclcpp::init.
 Waiting to give an opportunity for interrupt...
-[INFO] [rclcpp]: signal_handler(signal_value=2)
```

Comment:
This line has some value in this test, imo... Maybe instead all of the messages being printed should also be using console logging.

Reply:
After thinking about it further, I agree.  The latest changes here restore it and change everything to using `RCLCPP_INFO`.